cli/init.js

```diff
@@ -3,21 +3,38 @@
 const fs = require('fs')
 const path = require('path')
 
-const { MOULD_DIRECTORY } = require('./constants')
+const {
+    MOULD_DIRECTORY,
+    RESOLVERS,
+    SYMLINK_MOULD_DIRECTORY,
+} = require('./constants')
 
+const appPath = path.join(__dirname, '..')
 const originalDirectory = process.cwd()
+
 const mouldPath = path.join(originalDirectory, MOULD_DIRECTORY)
+const resolversPath = path.join(mouldPath, RESOLVERS)
+const symlinkMouldPath = path.join(appPath, SYMLINK_MOULD_DIRECTORY)
 
 if (fs.existsSync(mouldPath)) {
     console.warn(
-        `You already have ${MOULD_DIRECTORY} initialized at ${originalDirectory}\n`
+        `You already have ${MOULD_DIRECTORY} initialized at ${originalDirectory}`
     )
 } else {
     fs.mkdirSync(mouldPath)
 
-    console.log(
-        `Created ${MOULD_DIRECTORY} directory at ${originalDirectory}\n`
-    )
+    console.log(`Created ${MOULD_DIRECTORY} directory at ${originalDirectory}`)
+}
+
+if (!fs.existsSync(resolversPath)) {
+    fs.writeFileSync(resolversPath, 'export default {}')
+
+    console.log(`Created ${RESOLVERS} at ${mouldPath}`)
+}
+
+if (fs.existsSync(symlinkMouldPath)) {
+    fs.unlinkSync(symlinkMouldPath)
 }
+fs.symlinkSync(mouldPath, symlinkMouldPath, 'dir')
```

Comment:
Create symlink on `mould init` may met this situation: A had ran `mould init` and committed. B pulled code, ran `mould dev` but there's no /.mould directory that was prepared. What about doing this on `mould dev`

Reply:
Good catch, thank for mentioning that. I'll move synlinking into  the `dev` script